utils/near.js

```diff
@@ -0,0 +1,51 @@
+const nearAPI = require('near-api-js');
+
+const creatorKeyJson = (() => {
+    try {
+        return JSON.parse(process.env.ACCOUNT_CREATOR_KEY);
+    } catch (e) {
+        console.warn(`Account creation not available.\nError parsing ACCOUNT_CREATOR_KEY='${process.env.ACCOUNT_CREATOR_KEY}':`, e);
+        return null;
+    }
+})();
+
+const keyStore = {
+    async getKey() {
+        return nearAPI.KeyPair.fromString(creatorKeyJson.secret_key || creatorKeyJson.private_key);
+    },
+};
+
+const nearPromise = (async () => {
+    const near = await nearAPI.connect({
+        deps: { keyStore },
+        masterAccount: creatorKeyJson && creatorKeyJson.account_id,
+        nodeUrl: process.env.NODE_URL
+    });
+    return near;
+})();
+
+const getContract = async (contractName, viewMethods, changeMethods, secretKey) => {
```

Comment:
where is `getContract` called?

Reply:
In an upcoming PR. For now it's just a helper method, shouldn't really be an issue to include it here.

Reply:
Let's include it when it's actually used. E.g. it probably shouldn't use `creatorKeyJson` at all. 

Same actually might apply to `withNear` as well. Looks like we ideally should depend on `creatorKeyJson` only in middleware which is using it to create accounts.